wordpress-mobile/WordPress-Android
# Add custom selector for password button

Fixes https://github.com/wordpress-mobile/WordPress-Android/issues/9905

**Summary:**
As we have struggled with google-material-library `1.1.0` and above, we have picked @oguzkocer suggestion https://github.com/wordpress-mobile/WordPress-Android/issues/9905#issuecomment-577979372 to use custom drawable and not to increment google-material-library. 

If this solution doesn't make an impact, we will try some other approach.

To test: 
1. Open the app
2. Make sure you are logged out
3. Go to the login screen with the password input field
4. Play around with the password (eye) icon.

PR submission checklist:
